projects/microphysics/configs/gscond-only.yaml

```diff
@@ -1,13 +1,13 @@
 # submit with
-# python3 argo/end_to_end.py train/precpd-only.yaml
+# python3 argo/end_to_end.py configs/gscond-only.yaml
 kind: prognostic
-experiment: pscaling-test
-name: gscond-only-tscale-zcloud-retrained-v1
+experiment: gscond-precpd
```

Comment:
Setting this script back to the better performing old model vs. the more recent retrained version.